package.json

```diff
@@ -19,12 +20,13 @@
   },
   "homepage": "https://github.com/redfieldstefan/msv2",
   "dependencies": {
-    "body-parser": "^1.13.3",
-    "express": "^4.13.3",
-    "mongoose": "^4.1.5"
+    "body-parser": "1.13.3",
```

Comment:
Let's grab specific version numbers from now on (no `^`), just to guarantee consistency between installs.